docs/conversion_example_gallery.rst

```diff
@@ -61,6 +73,7 @@ Segmentation
 
 Behavior
 --------
+
```

Comment:
```suggestion
Pose estimation
~~~~~~~~~~~~
```

?

Reply:
Would you also want to put together a gallery + named section for the general purpose MovieInterface (since there are examples files available through the fixtures anyway)?

Reply:
Yes, I think that's something to add.

Reply:
we can do that later once there is more than one interface here